src/clientSideScene/ClientSideSceneComp.tsx

```diff
@@ -4,6 +4,10 @@ import toast from 'react-hot-toast'
 
 import type { Node } from '@rust/kcl-lib/bindings/Node'
 
+// Helper function to check if overlays should always be shown
+const shouldAlwaysShowOverlays = () =>
+  localStorage.getItem('showAllOverlays') === 'true'
+
```

Comment:
@franknoirot I added logic for turning all of the overlays at the start of a test, I have not migrated any existing tests over to this though yet.